buildflags/buildflags.gni

```diff
@@ -33,4 +33,7 @@ declare_args() {
 
   # Enable Spellchecker support
   enable_builtin_spellchecker = true
+
+  # Undocumented Windows dark mode API
+  enable_win_dark_mode = true
```

Comment:
Is this supposed to be enabled by default?

Reply:
Disbled by default in 8508c62af5a9015dfebc9d38d68486a763b70aa6